### DISCUSSION TITLE
Error: Unable to parse JSON response

### DISCUSSION THREAD

**Discussion by tictac22** [wintercms/winter#489] [Q&A] [answered]
Hi! The problem I have is the installation of cms. I get such error:
![image](https://user-images.githubusercontent.com/70350446/157521360-1518e6ae-9fff-46cc-a5e9-f18790f10597.png)

I've already set in `php.ini` `max_input_time = 500`. What else I can check to remove this error

**Comment by tictac22** [comment, marked as answer]
MAINTAINER EDIT: The issue was the result of PHP 8.1 being installed. Until the Laravel 9 release of Winter CMS is made available, we are unable to support PHP 8.1. More information on this upcoming release can be found here: https://github.com/wintercms/winter/pull/148

How `max_input_time` looks in `php.ini`
![image](https://user-images.githubusercontent.com/70350446/157523480-57a130e2-f21c-40a0-8bb7-001fb88ab5ca.png)

**Reply by LukeTowers** [reply]
Check your browser dev console to see what response is being sent by your server that is triggering the error.

**Reply by tictac22** [reply]
I don't get any response.

![image](https://user-images.githubusercontent.com/70350446/157525428-64debf82-35ad-44ab-9fbe-ee654b722af9.png)

**Reply by LukeTowers** [reply]
What about in the headers?

**Reply by tictac22** [reply]
Smth like this
![image](https://user-images.githubusercontent.com/70350446/157526529-e98e40a7-8397-4031-a4b1-8ed20671eac7.png)

**Reply by LukeTowers** [reply]
You can't run the installer on PHP 8.1 right now, that would require using the wip/1.2 branch. @bennothommo can we fix the PHP version check in the installer?

@tictac22 just switch the version of PHP used by your webserver to 8.0 and you should be able to run the installer.

**Reply by tictac22** [reply]
Wow it helped me thanks a lot!

**Reply by bennothommo** [reply]
A new release of the Winter CMS installer is now available with a check for the upper PHP versions: https://github.com/wintercms/web-installer/releases/latest/download/install.zip

**Comment by gergo85** [comment]
It doesn't work :(

PHP version: 8.1.9 and 8.0.22  
json support | enabled

**Reply by bennothommo** [reply]
@gergo85 do you have an `install.log` file I can take a look at? It should be in your install folder when you ran the installer.

**Reply by gergo85** [reply]
Thanks for your quick reply! How can I send the file?

**Reply by bennothommo** [reply]
@gergo85 it's fine for you to upload here, or you can send it through to us on Discord. (https://discord.gg/D5MFSPH6Ux)

**Comment by gergo85** [comment]
I imported the DB to server from localhost, and I got a following error:

![Képkivágás](https://user-images.githubusercontent.com/2959112/219973209-cdb2ef88-104f-4948-a804-f5f84ecd11b2.JPG)

**Reply by bennothommo** [reply]
@gergo85 which MySQL version are you running?

**Reply by gergo85** [reply]
5.5.68-MariaDB

**Reply by LukeTowers** [reply]
You'll need to update, see https://wintercms.com/docs/setup/installation#system-requirements

**Reply by gergo85** [reply]
Thank you very much for the quick reply. I wrote to the hosting provider to update it.

**Reply by LukeTowers** [reply]
No problem, good luck!
